modules/ROOT/nav.adoc

```diff
@@ -38,7 +38,7 @@
  *** xref:proxy-deploy-runtime-fabric.adoc[Deploy an API Proxy to Runtime Fabric]
  ** xref:download-proxy-task.adoc[Downloading an API Proxy]
  ** xref:proxy-domain-support.adoc[Automatic Domain Support for Mule 4 API Proxies]
- ** xref:proxy-mule3-elements.adoc[Mule 3.x API Proxy Elements]
```

Comment:
I'd prefer Proxy Elements or something in the same spirit. Doing that, will allow to add additional descriptions of stuff used on Mule 3 proxies, like WSDL functions, for example.

Reply:
I think creating a bucket to include all content is not a great way to organize content. When we need to add new content, we can redetermine. One thing to keep in mind is that topic titles should give a glimpse into what the topic talks about. Descriptive and specific headings are a requirement for good documentation. Hope that makes sense! Thanks!

Reply:
Ok. Take also into account that renaming a doc each time that we need to add content not always is a good thing. Also, we already walked the path of having micro articles with not so good results in general.